### PR TITLE
better docs

### PR DESCRIPTION
**Pull request by Zamiell**

### CODE REVIEW

File: docs/build-instructions-electron.md [hunk 1]
```diff
@@ -115,11 +116,30 @@ initialized, Steam will prevent the user from opening that game, saying that
...
+copy node_modules\greenworks\samples\electron\main.js renderer.js
 ```
 
 11) Test it:
 
 ```shell
 npm start
 ```
+
+## A Note on electron-builder
```
Comment (hokein) [inline docs/build-instructions-electron.md]: This is duplicated with `Using Greenworks with Electron` in `gotchas.md`, I think keeping it in one place is sufficient, so just remove this one.

File: docs/gotchas.md [hunk 2]
```diff
@@ -1,16 +1,39 @@
...
 is initialized, Steam will prevent the user from opening that game, saying that
 "that game is already open".
+
+## Using Greenworks with Multi-Platform Projects
+
+* Greenworks is a native node addon. When you compile it, the binary will only
+for your OS. This means that if you want to build Greenworks for all 3 platforms
+(Windows, Mac, Linux), you need to setup 3 different platforms.
+
+## Using Greenworks with Electron
```
Comment (hokein) [inline docs/gotchas.md]: I'd mention `electron-builder` specifically, how about rephrasing to `Using Greenworks with electron-builder`?

**Comment by hokein** [comment]
@patrickklug Does the change in readme.md look good to you?

**Comment by Zamiell** [comment]
ok

**Comment by hokein** [comment]
The PR looks good to me now. I'm merging it now. @patrickklug Let me know if you have post comments on it. 

@Zamiell Thanks for the contribution!

**Pull request merged**